getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -9,15 +9,15 @@ importing the start assets and setting up the game scene.
 We've prepared a Godot project with the 3D models and sounds we'll use for this
 tutorial, linked in the index page. If you haven't done so yet, you can download
 the archive here: `Squash the Creeps assets
-<https://github.com/godotengine/godot-3d-dodge-the-creeps/releases/tag/1.1.0>`__.
```

Comment:
This would require https://github.com/godotengine/godot-docs-project-starters/pull/9 to be reviewed and merged first, or another location that i should reference to make this doc valid for 4.x